### PR TITLE
Update glueviz package to 1.2.4

### PR DESCRIPTION
**Pull request by ryanskeith**
Turning glueviz into a metapackage like pypi on pypi.org.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -5,35 +5,24 @@
...
-source:
-  fn: glueviz-{{version}}.tar.gz
-  url: https://pypi.io/packages/source/g/glueviz/glueviz-{{version}}.tar.gz
-  sha256: 89db2b2331eb74f221251b129f6ce6b507f14101caea993ee0926ceeeaaf68ab
-
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  # Skipping unnecessary linux platforms that don't need viz.
+  skip: True  # [py<36 or ppc64le or s390x]
```
Comment (marcoesters) [inline recipe/meta.yaml]: glue-core requires python>=3.7: https://github.com/glue-viz/glue/blob/v1.2.4/setup.cfg#L26

File: recipe/meta.yaml [hunk 2]
```diff
@@ -5,35 +5,24 @@
...
 
 requirements:
-  build:
-    - python
-    - pip
-    - setuptools >=30.3
-    - setuptools_scm
+  is_meta_pkg: True
```
Comment (cbouss) [inline recipe/meta.yaml]: I don't see this documented in conda build documentation, but it doesn't hurt.

**Review by cbouss — COMMENTED** [review]
Good to go, if the skip is updated as Marco highlighted.

File: recipe/meta.yaml (hunk 2 above)
Comment (ryanskeith) [reply recipe/meta.yaml]: This was taken from our Anaconda Distribution.  I think you are correct that it doesn't matter.

**Review by marcoesters — APPROVED** [review]
LGTM

**Review by AndrewVallette — APPROVED** [review]
LGTM!

**Pull request merged**